apps/issuer/api.py

```diff
@@ -99,6 +99,59 @@ def put(self, request, **kwargs):
     def delete(self, request, **kwargs):
         return super(IssuerDetail, self).delete(request, **kwargs)
 
+class CollectionBadgeDetail(BaseEntityDetailView):
+    model = CollectionBadgeContainer
+    v1_serializer_class = CollectionBadgeClassSerializerV1
+    # v2_serializer_class = CollectionBadgeClassSerializerV2
+    permission_classes = (AuthenticatedWithVerifiedIdentifier, AuditedModelOwner)
+    valid_scopes = {
+        'get': ['r:issuer', 'rw:issuer'],
+        'post': ['rw:issuer'],
+        'put': ['rw:issuer'],
+        'delete': ['rw:issuer']
+    }
+
+    @apispec_get_operation('CollectionBadge',
+                           summary='Get a single CollectionBadge',
+                           tags=['Issuer']
```

Comment:
Why is the tag `Issuer` and not something referring to collection badges?
This also applies to all the cases below.

Reply:
Updated the tags to ``` CollectionBadges```